polymer/index.html

```diff
@@ -0,0 +1,16 @@
+<head>
+  <base href="https://polygit.org/polymer+v2.2.0/components/">
+  <script src="webcomponentsjs/webcomponents-lite.js"></script>
+  <script src="scripts/index.js"></script>
+  <link rel="import" href="polymer/polymer.html">
+  <link rel="stylesheet" href="/styles/main.css">
+</head>
+<body>
+  <x-foo></x-foo>
+
+  <dom-module id="x-foo">
+    <template>
+      <span>{{foo}}</span>
+    </template>
+  </dom-module>
+</body>
```

Comment:
Wrap it in 'html' tag. Validate here - https://validator.w3.org/#validate_by_input